src/TaylorSeries.jl

```diff
@@ -32,7 +32,7 @@ export get_coeff, derivative, integrate,
     show_params_TaylorN,
     get_order, get_numvars,
     set_variables, get_variables,
-    ∇, jacobian, hessian
+    ∇, jacobian, jacobian!, hessian
```

Comment:
I guess the same functionality can also be added for `hessian`; can you also implement it?